lang/en/docs/tutorials/materials/specific/defect_point_interstitial_tin_oxide.md

```diff
@@ -0,0 +1,138 @@
+---
+# YAML header
+render_macros: true
+---
+
+# O-interstitial Defect in SnO
+
+## Introduction
+
+This tutorial demonstrates how to create an oxygen interstitial defect in tin monoxide (SnO), following the methodology described in the literature.
```

Comment:
We should mention that voronoi site is used as a placement method and maybe add a link to the implementation and/or more information